README.md

```diff
@@ -561,7 +574,7 @@ romcal.queryFor(dates, {
 
 ## Overriding dates <a name="overridingDates"></a>
 
-romcal has been designed with extensibility in mind to cater for unique scenarios that are common in the liturgical calendar.
+romcal was designed with extensibility in mind to cater for unique scenarios that are common in the liturgical calendar.
```

Comment:
Same here

Reply:
Now I think that it would be even better expressed by using present tense also here. What do you think?

`romca is designed with ...`